tests/integration/assessment/test_workflows.py

```diff
@@ -33,14 +32,19 @@ def test_running_real_assessment_job(
     tmp_table = installation_ctx.make_table(schema_name=source_schema.name, ctas="SELECT 2+2 AS four")
     view = installation_ctx.make_table(schema_name=source_schema.name, ctas="SELECT 2+2 AS four", view=True)
     non_delta = installation_ctx.make_table(schema_name=source_schema.name, non_delta=True)
-
-    installation_ctx.__dict__['include_object_permissions'] = [f"cluster-policies:{cluster_policy.policy_id}"]
+    installation_ctx = installation_ctx.replace(
+        config_transform=lambda wc: dataclasses.replace(
+            wc,
+            include_object_permissions=[f"cluster-policies:{cluster_policy.policy_id}"],
+            include_databases=[source_schema.name],
+        ),
+    )
     installation_ctx.workspace_installation.run()
-
     populate_for_linting(installation_ctx.installation)
```

Comment:
Make this a proper fixture or add it to the context. Then reuse it in the external hivemetastore and migration progress tests

Reply:
IIRC a fixture doesn't work because you can't upload data before the installation is complete

Reply:
See the changes, the fixture did not following the pattern of the other fixture. Moreover it did not set the `include_job_ids` and `include_dashboard_ids` in the config correctly, thus the linter ran for all jobs and dashboards.